Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -137,6 +141,9 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     private func loadEditor() {
         if let editorURL = editorURL ?? ProcessInfo.processInfo.environment["GUTENBERG_EDITOR_URL"].flatMap(URL.init) {
             webView.load(URLRequest(url: editorURL))
+        } else if plugins,
+                  let editorURL = URL(string: "\(siteURL)/wp-content/plugins/block-editor-assets-endpoint/editor/remote.html") {
```

Comment:
I'm still going through the PR and changes to gain the full context but can you explain a bit more why we are loading the remote `remote.html` file from the plugin? If we have this locally, can't we load the local version?

Reply:
I took the approach of loading a remote `remote.html` as I encountered CORS errors fetching the API endpoint. My understanding is that WordPress defaults to a lax [CORS policy](https://developer.wordpress.org/rest-api/frequently-asked-questions/#why-is-the-rest-api-not-verifying-the-incoming-origin-header-does-this-expose-my-site-to-csrf-attacks). So, we should, in theory, be able to perform the API endpoint request. 

The CORS errors were seemingly sporadic. The errors might've related to a given site's host, as I imagine different hosts could tighten CORS policies. I may have also been fighting unexpected caching that confused my testing. 

Nonetheless, I'll retest whether CORS is a legitimate issue for this feature.

Reply:
From testing now, there does not appear to be CORS errors when leveraging the local `remote.html` file. I updated the code to use the local copy in https://github.com/wordpress-mobile/GutenbergKit/pull/22/commits/179c3e7868212faca4b55b7c94998ed0be796c9f.